extensions/resource-deployment/package.json

```diff
@@ -43,6 +43,12 @@
       }
     ],
     "menus": {
+      "commandPalette": [
+        {
+          "command": "azdata.resource.deploy",
```

Comment:
what's this for?

Reply:
to hide the generic command from command palette. this is what we decided to do during the meeting with Dinakar.